### ISSUE TITLE
Delegate Proposals

### ISSUE THREAD

**Issue by Jarunik** [ArkEcosystem/ark-explorer#30]
https://github.com/ArkEcosystem/ark-explorer/blob/master/lib/api/delegates.js

I think /api/delegates/getDelegateProposals does not really do anything anymore as it is still pointing to lisk.

Maybe we should remove it?

**Comment by 4miners** [comment]
@Jarunik Yep, it's currently disabled by default. It should be adapted to Ark or removed entirely.
